### ISSUE TITLE
Add Metric "Retr"

### ISSUE THREAD

**Issue by Technikte** [xsteadfastx/iperf3exporter#2]
Maybe, it is possible to include the "Retr" value in the metrics ?

> In iperf3 the column Retr stands for Retransmitted TCP packets and indicates the number of TCP packets that had to be sent again (=retransmitted).

Example
```
                "sum_sent":     {
                        "start":        0,
                        "end":  5.038733959197998,
                        "seconds":      5.038733959197998,
                        "bytes":        337025600,
                        "bits_per_second":      535095685.11316043,
                        "retransmits":  7,
                        "sender":       false
                },
```
![image](https://user-images.githubusercontent.com/49494924/142424124-f643b20f-74eb-4925-b1d0-914ee2efc3e9.png)

**Comment by xsteadfastx** [comment]
i will try to add this today.

**Comment by xsteadfastx** [comment]
feature is in 0.3.0 release. thanks for this issue. included through commit de76efa1b569a61a84b9c9345bc43b708cdba608 i would love to hear more from users who actually using this and have more feature requests.
